game-core/src/main/java/games/strategy/engine/lobby/client/login/LobbyPropertyFileParser.java

```diff
@@ -24,17 +27,26 @@
  */
 class LobbyPropertyFileParser {
 
+  @VisibleForTesting static final String YAML_HOST = "host";
```

Comment:
I updated this, but I regret it. The line length on commentary is going to cause issues anyways.  Let's not worry about guava formatting vs our custom formatting for the moment in PRs please and follow-up in: https://github.com/triplea-game/triplea/issues/4488

Reply:
I think our goal should be that any formatting problems are auto-flagged as build problems. It's the point of checkstyle IMO to avoid reviewing for that problem. To one extent, if something passes through checkstyle, it's good to go. From the point of view of code-format-ping-pong, there is a problem. Hence I'm leaning for us to solve this with a deterministic/strong formatter that AFAIK always produces identical output. We can wire that in the build to both run formatting and to enforce formatting.